data.json

```diff
@@ -599,4 +599,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/29686102/106919215-f0711a00-672f-11eb-88f6-8ab44f67b911.png",
     "issueNumber": 313
   }
+  {
+    "name": "Utkarsh Mishra",
+    "githubUsername": "Utkarsh1504",
+    "imageUrl": "https://user-images.githubusercontent.com/76392681/107881996-f673b180-6f0c-11eb-90a8-9365ff3e4753.png",
+    "issueNumber": 361
+},
```

Comment:
- Indentation is not quite right 
- Last object does not need a `,`

Thank you for the contribution, just a few small tweaks, then we can get it merged in 👍

Reply:
Thank you for your guidance. Now I have fixed the indentation.

Reply:
Awesome, thank you 👍